### PR TITLE
Node crud

### PR DESCRIPTION
**Pull request by ljhaywar**
@schmalliso This one is ready for you to review.  As we discussed, the REFs aren't currently working.

### CODE REVIEW

File: source/quickstart/node-crud-tutorial.txt [hunk 1]
```diff
@@ -0,0 +1,1037 @@
...
+
+.. pubdate:: October 23, 2019
+
+.. updated-date:: March 4, 2020
+
+.. type:: quickstart
+
+.. level:: beginner
+
+.. tags::
```
Comment (MaBeuLux88) [inline source/quickstart/node-crud-tutorial.txt]: Remove the tags, they don't exist

File: source/quickstart/node-crud-tutorial.txt [hunk 2]
```diff
@@ -0,0 +1,1037 @@
...
+.. level:: beginner
+
+.. tags:: 
+
+   * node
+   * javascript
+
+.. languages::
+
+   * nodejs
```
Comment (MaBeuLux88) [inline source/quickstart/node-crud-tutorial.txt]: Replace by "nodejs" by "Javascript"

File: source/quickstart/node-crud-tutorial.txt [hunk 3]
```diff
@@ -0,0 +1,1037 @@
...
+
+.. products::
+
+   * MongoDB
+   * Atlas
+
+.. atf-image:: /images/hero-node.png
```
Comment (MaBeuLux88) [inline source/quickstart/node-crud-tutorial.txt]: Replace atf image by this one: /images/atf-images/quickstart/nodejs.png
It's the correct size. Only use images for ATF from the ATF folder.

File: source/quickstart/node-crud-tutorial.txt [hunk 4]
```diff
@@ -0,0 +1,1037 @@
...
+
+.. related::
```
Comment (MaBeuLux88) [inline source/quickstart/node-crud-tutorial.txt]: Replace this by a Node Series in snooty.toml like this one: 

"Java Tutorials" = ["/quickstart/java-setup-crud-operations", "/quickstart/java-mapping-pojos", "/quickstart/java-aggregation-pipeline", "/quickstart/java-change-streams"]

File: source/quickstart/node-crud-tutorial.txt [hunk 5]
```diff
@@ -0,0 +1,1037 @@
...
+   * `How to connect to a MongoDB database using Node.js <https://www.mongodb.com/blog/post/quick-start-nodejs-mongodb--how-to-get-connected-to-your-database>`__
+   * `Video: How to perform the CRUD operations using MongoDB & Node.js <https://www.mongodb.com/blog/post/quick-start-video-nodejs--mongodb--the-crud-operations>`__
+   * `How to analyze your data using MongoDB’s Aggregation Framework and Node.js <https://www.mongodb.com/blog/post/quick-start-nodejs--mongodb--how-to-analyze-data-using-the-aggregation-framework>`__
+   * `How to implement transactions using Node.js <https://www.mongodb.com/blog/post/quick-start-nodejs--mongodb--how-to-implement-transactions>`__
+
+.. ---------------------------------------------------------------
+.. ARTICLE CONTENT
+.. ---------------------------------------------------------------
+
+===============================================
```
Comment (MaBeuLux88) [inline source/quickstart/node-crud-tutorial.txt]: Remove one equal sign, it's too long.

File: source/quickstart/node-crud-tutorial.txt [hunk 6]
```diff
@@ -0,0 +1,1037 @@
...
+MongoDB and Node.js Tutorial - CRUD Operations
+===============================================
+
+.. introduction::
+
+    .. image:: /images/quickstart-node.png
+      :alt: QuickStart for Node.js Logo
+      :scale: 50%
```
Comment (MaBeuLux88) [inline source/quickstart/node-crud-tutorial.txt]: 30%

File: source/quickstart/node-crud-tutorial.txt [hunk 7]
```diff
@@ -0,0 +1,1037 @@
...
+      :align: right
+
+    `In the first post in this
+    series <https://www.mongodb.com/blog/post/quick-start-nodejs-mongodb--how-to-get-connected-to-your-database>`__,
```
Comment (MaBeuLux88) [inline source/quickstart/node-crud-tutorial.txt]: Replace all the >`__ by >`_.
Our links only have one underscore.

File: source/quickstart/node-crud-tutorial.txt [hunk 8]
```diff
@@ -0,0 +1,1037 @@
...
+    ======================== ============
+    Component                Version Used      
+    ======================== ============  
+    MongoDB                  4.0  
+    MongoDB Node.js Driver   3.3.2  
+    Node.js                  10.16.3   
+    ======================== ============
+
+
+    All Posts in the *Quick Start: Node.js and MongoDB* Series:
```
Comment (MaBeuLux88) [inline source/quickstart/node-crud-tutorial.txt]: Delete. The series will make a nice menu at the bottom.

File: source/quickstart/node-crud-tutorial.txt (hunk 1 above)
Comment (ljhaywar) [reply source/quickstart/node-crud-tutorial.txt]: Added Node.js to the tags list

File: source/quickstart/node-crud-tutorial.txt (hunk 4 above)
Comment (ljhaywar) [reply source/quickstart/node-crud-tutorial.txt]: Waiting to create the series until all posts are ported to the Dev Hub

**Pull request merged**